app/routes/meetings/MeetingCreateRoute.ts

```diff
@@ -26,8 +26,8 @@ const mapStateToProps = (state, props) => {
   return {
     user: props.currentUser,
     initialValues: {
-      startTime: time(17, 15),
-      endTime: time(20),
+      startTime: time(16, 15),
+      endTime: time(18),
```

Comment:
I see that both Google Calendar and Outlook sets the "default" time to the current time (or rather the closest half hour). Could we perhaps do something similar?

Reply:
Sure, I think 15 minutes past the next hour would be appropriate for our users. Although I am very much open to suggestions.

Reply:
After discussing it at the office we concluded that a hard coded time suited our needs better

Reply:
Why? Simply because _we_ have meetings at 16?